README.md

```diff
@@ -1,7 +1,7 @@
 # Viam Triton ML Model Service
 
 A Viam provided ML Model service backed by NVIDIA's [Triton Inference Server](https://developer.nvidia.com/triton-inference-server).
-Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) to deploy TensorFlow or PyTorch ML models to your machine faster while consuming less computer power.
+Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) to deploy TensorFlow (not TensorFlow Lite) or PyTorch ML models to your machine faster while consuming less computer power.
```

Comment:
You can use Triton server just fine on platforms that aren't Jetson (though that wasn't always true), or even that don't have an NVidia GPU (though it will only be able to use the CPU for inference).

Reply:
```suggestion
Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) or another supported machine to deploy [supported models](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/backend/docs/backend_platform_support_matrix.html) to your machine faster while consuming less computer power.
```